README.md

```diff
@@ -63,6 +63,21 @@ kubectl create secret generic snyk-monitor -n snyk-monitor --from-file=./dockerc
 kubectl create configmap snyk-monitor-certs -n snyk-monitor --from-file=<path_to_certs_folder>
 ```
 
+5. If you are using an insecure registry or your registry is using unqualified images, you can provide a `registries.conf` file. See [the documentation](https://github.com/containers/image/blob/master/docs/containers-registries.conf.5.md) for information on the format and examples.
+
+Create a file named `registries.conf`, see example adding an insecure registry: 
+
+```
+[[registry]]
+location = "internal-registry-for-example.net/bar"
+insecure = true
+```
+
+Once you've created the file, you can use it to create the following ConfigMap:
+```shell
+kubectl create configmap snyk-monitor-registries-conf -n snyk-monitor --from-file=./registries.conf
```

Comment:
Is this `file=./registries.conf` the local file path? If it is, what do you think to make it explicit `file=/local/path/registries.conf`? Not sure if it helps, but I'm always confused with these things when I read documentation.

Reply:
great point, I suggest to be consistent with the rest of the docs. in the ssl certs part I mentioned the path as `--from-file=<path_to_certs_folder>`, it is best to to the same here, like `--from-file=<path_to_registies_conf_filer>`

Reply:
Sounds good, I was trying to be consistent with the docker conf instruction, but I think that makes more sense :-)